packages/fxa-settings/src/components/UnitRow/index.stories.tsx

```diff
@@ -59,4 +60,32 @@ storiesOf('Components|UnitRow', module)
         </UnitRow>
       );
     }
-  );
+  )
+  .add('with modal-triggering secondary CTA', () => {
+    const [modalRevealed, revealSecondaryModal, hideModal] = useBooleanState();
```

Comment:
I think we should probably call these `[secondaryModalRevealed, revealSecondaryModal, hideSecondaryModal]` just so it's clear these are all for the modal tied to the secondary CTA.